package.json

```diff
@@ -17,17 +17,16 @@
     "url": "https://github.com/unumotors/mongoose-harakiri/issues"
   },
   "homepage": "https://github.com/unumotors/mongoose-harakiri#readme",
-  "dependencies": {
-    "debug": "^3.1.0"
-  },
   "peerDependencies": {
-    "mongoose": "4.x"
+    "mongoose": "^5.2.5"
   },
   "devDependencies": {
     "ava": "^0.25.0",
-    "eslint": "^5.0.1",
+    "mongoose": "^5.2.5",
+    "eslint": "^4.19.1",
     "eslint-config-unu": "^1.2.2",
     "eslint-plugin-import": "^2.13.0",
-    "mongoose": "4.13.14"
+    "eslint-friendly-formatter": "^3.0.0",
+    "eslint-plugin-vue": "^4.7.1"
```

Comment:
why do we have this here?

Reply:
could you make use of unu's default eslint config?

Reply:
These are peer dependencies of eslint-config-unu:
```
npm WARN eslint-config-unu@1.2.2 requires a peer of eslint-friendly-formatter@^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-unu@1.2.2 requires a peer of eslint-plugin-vue@^4.3.0 but none is installed. You must install peer dependencies yourself.
```
See: https://github.com/unumotors/eslint-config-unu/blob/master/package.json

Reply:
@pelzerim then let's still remove them from here but add them as normal dependencies to eslint-config-unu. it doesn't make sense to keep them here.

in addition, we can create eslint-config-unu-vue to have these dedicated dependencies and keep eslint-config-unu pure.

Reply:
Let's put them as normal dependencies and see how it goes :)